### ISSUE TITLE
REPL log output is not displayed when inside async function

### ISSUE THREAD

**Issue by jaydson** [6to5/6to5.github.io#15]
Odd.
It seems the `console.log` is being ignored by the REPL output when we call it inside an async function.  
I took too long to figure out the code was working but just was not displaying the log in the REPL.
In the browser dev tools it works well, i mean, the log appear.

The code i was trying:
```javascript
function p1() {
  return new Promise(function(resolve, reject){
    setTimeout(function() {
      resolve('p1');
    }, 300);
  });
}

async function getVal() {
  var val1 = await p1();
  return val1;
}

(async function() {
  var bla = await getVal();
  console.log(bla); 
}());
```

![screenshot from 2015-01-07 11 44 00](https://cloud.githubusercontent.com/assets/72206/5646400/cab0bd88-9662-11e4-98f8-ebe322693c6b.png)

**Comment by sebmck** [comment]
Current REPL implementation just binds `console.log`, `eval`s the transformed code, then dumps the console buffer.

@thejameskyle Any ideas on ways to improve this?

**Comment by thejameskyle** [comment]
Fixed by #17, thanks for pointing this out!

**Comment by jaydson** [comment]
:+1:
